inspec.yml

```diff
@@ -365,6 +369,4 @@ inputs:
       - "V-93567" # SI-02(02)
       - "V-93563" # SI-16
       - "V-93565" # SI-16
```

Comment:
Add to Low baseline:
      - "V-92975" # AC-02(02)
      - "V-92977" # AC-02(02)

Reply:
Done! Good catch - thanks!
[Commit reference](https://github.com/CMSgov/cms-ars-5.0-microsoft-windows-server-2019-stig-overlay/pull/2/commits/d784d78edf55bd449b351557aee0acd333f8fe98)